projects/storefrontlib/src/cms-components/user/login-form/login-form.component.ts

```diff
@@ -37,22 +37,15 @@ export class LoginFormComponent implements OnInit, OnDestroy {
       password: ['', Validators.required],
     });
 
-    if (
-      this.checkoutConfigService &&
-      this.checkoutConfigService.isGuestCheckout()
-    ) {
-      this.loginAsGuest = this.activatedRoute.snapshot.queryParams['forced'];
+    if (this.checkoutConfigService.isGuestCheckout()) {
+      this.loginAsGuest = this.activatedRoute?.snapshot?.queryParams?.['forced'];
     }
 
-    // TODO(issue:#4055) Deprecated since 1.1.0
-    if (this.winRef && this.winRef.nativeWindow) {
-      const routeState =
-        this.winRef.nativeWindow.history &&
-        this.winRef.nativeWindow.history.state;
+    const routeState = this.winRef?.nativeWindow?.history?.state;
```

Comment:
Could you change that to prefilledEmail and move the last param to the chained properties?

Reply:
done